src/js/components/Select/stories/Clear.js

```diff
@@ -2,7 +2,10 @@ import React, { useState } from 'react';
 
 import { Box, Select } from 'grommet';
 
-const options = ['one', 'two', 'three'];
+const options = [];
+for (let i = 0; i < 500; i += 1) {
+  options.push(`Number ${i}`);
+}
 
 const ClearTop = () => {
   const [value, setValue] = useState();
```

Comment:
can we add `dropHeight="large"` or something similar?

Reply:
Done.